bin/resin-preload

```diff
@@ -2,9 +2,12 @@
 
 'use strict'
 
-const fs = require('fs')
+const Docker = require('dockerode')
 const preload = require('..')
 const info = require('../package.json')
+const streamToPromise = require('stream-to-promise')
+
+const docker = new Docker({Promise})
```

Comment:
That's only for the standalone bin/resin-preload . I didn't change it.
The new resin-cli preload way allows you to specify a docker host or socket.